docs/tutorials/enable-totp-openstack.md

```diff
@@ -0,0 +1,80 @@
+# Configuring Time-Based One-Time Password (TOTP) on OpenStack
+
+Author: Ramon Grullon
+
+## Introduction
+
+Horizon added time-based one-time password (TOTP) authentication support,
+leveraging the already existing two factor authentication from Keystone.
+Now, if a user activates TOTP on Keystone, it gets activated on Horizon too.
+
+## Prerequisites
+
+- A working OpenStack 2023.2 environment deployed using kolla-ansible.
+- Enabled TOTP on all control nodes
+- Admin access to the OpenStack dashboard and CLI.
+- Access to the kolla-ansible configuration files.
+
```

Comment:
To the Prerequisites, mention Keystone needing to be reconfigured, something like:
- [Keystone reconfigured with TOTP enabled](https://docs.openstack.org/keystone/latest/admin/auth-totp.html#configuring-totp)